modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/internal/upgrade/RemoteAppServiceUpgrade.java

```diff
@@ -29,6 +32,16 @@
 	public void register(Registry registry) {
 		registry.register(
 			"1.0.0", "1.0.1", new RemoteAppEntryURLUpgradeProcess());
+
+		registry.register(
```

Comment:
You can do all the modifications in the same process.

Reply:
Yes I know, but it is cleaner and better to split upgrades in small chunks, IMO (and in Chaparro's opinion IIRC :sweat_smile: )

Reply:
👏